tensorflow_datasets/core/dataset_builder.py

```diff
@@ -48,6 +53,15 @@
 
 import termcolor
 
+from typing import Any
```

Comment:
* Standard python imports should go before all third party imports. Please use pylint to check your code formatting: https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style
* Use `from typing import Any, Dict,...` rather than multi-line import
* Please sort `List`, `Dict`,... alphabetically

Reply:
Done